### ISSUE TITLE
Autoconnect on if credentials are saved

### ISSUE THREAD

**Issue by malaki86** [tzapu/WiFiManager#887]
I have a project that can use WiFi, but it's not required. What I'm looking for is a way to only attempt to autoconnect if there is already a WiFi connection saved. If not, skip it and the user can hit a button later on to start the AP config page.

I know that I can use the config page timeout, but I don't want the system to 'hang'.

**Comment by tablatronix** [comment]
You should be able to just getssid and check but its a little different in esp32

**Comment by SomeGuyNamedJay** [comment]
Does this work for you?
  if (WiFi.SSID()=="")
  {
    Serial.println("Blank");
  }
  else
  {
    Serial.println(WiFi.SSID());
    WiFi.begin(WiFi.SSID(), WiFi.psk());
  }

**Comment by malaki86** [comment]
I saw the getssid in the keywords file, but when I try to use it, I get a 'function not defined' error 

WiFi.SSID returns the said for the current, active WiFi connection. I need to read the saved one, if it exists.

**Comment by tablatronix** [comment]
Yeah that is the behavior on esp32 also, if you want to read the saved ssid you might have to read directly from the config struct.

```C++
    struct station_config conf;
    wifi_station_get_config_default(&conf);
    // conf.ssid;
```

**Comment by tablatronix** [comment]
on esp32 it is 

```C++
String WiFiManager::WiFi_SSID(){
  #ifdef ESP8266
    return WiFi.SSID();
  #elif defined(ESP32)
    //@todo , workaround only when not connected
    wifi_config_t conf;
    esp_wifi_get_config(WIFI_IF_STA, &conf);
    return String(reinterpret_cast<const char*>(conf.sta.ssid));
  #endif
}
```

**Comment by tablatronix** [comment]
Might be nice to make a getter for this

**Comment by malaki86** [comment]
It wouldn't even have to return the actual SSID or password that WiFiManager has saved. It could be a simple boolean return value.

Example:
  if WiFiManager.Saved == TRUE {
    WiFiManager.autoconnect;
  }
